apps/component-date/translations/src/en/fields.json

```diff
@@ -0,0 +1,10 @@
+{
+  "date-of-birth": {
+    "legend": "Date of birth",
+    "hint": "For example, 12 3 1980"
+  },
+  "passport-expiry": {
+    "legend": "Passport expiry date",
+    "hint": "For example, 30 04 2028"
```

Comment:
That's not a valid passport expiry date. Passports only last 10 years. I also don't care about this, I don't even know why I'm still typing.